Crash when replacing cables which are connected to Industrial revolution machines with cables from Powah

**Describe the bug**
Game crashed when I replaced some of the Techreborn energy cables with cables from Powah which were connected to some Industrial revolution machines

**Steps to Reproduce**
Steps to reproduce the behavior:
- Place some machine from Industrial Revolution
- Connect it to an energy source via Techreborn cables
- Try to replace these cables with Powah cables

**Screenshots**
I can't add screenshots because the game now is always crashing 

**Environment (please complete the following information with the version):**
 - Minecraft: 1.19.2
 - Mod Loader: fabric

**Logs**
[Log](https://pastebin.com/upvXCJA1)

**Additional context**

AE2 is indeed sending negative amounts.

```
[00:27:03] [Server thread/INFO]: [STDOUT]: sending power 4
[00:27:03] [Server thread/INFO]: [STDOUT]: sending power 8
[00:27:03] [Server thread/INFO]: [STDOUT]: sending power 4
[00:27:03] [Server thread/INFO]: [STDOUT]: sending power 4
[00:27:03] [Server thread/INFO]: [STDOUT]: sending power 8
[00:27:03] [Server thread/INFO]: [STDOUT]: sending power 4
[00:27:03] [Server thread/INFO]: [STDOUT]: sending power 10
[00:27:03] [Server thread/INFO]: [STDOUT]: sending power 16
[00:27:03] [Server thread/INFO]: [STDOUT]: sending power -8 <<< 
[00:27:03] [Server thread/INFO]: [STDOUT]: sending power -2 <<<
[00:27:03] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.class_148: Ticking block entity
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:879) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:819) ~[client-intermediary.jar:?]
	at net.minecraft.class_1132.method_3748(class_1132.java:98) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:665) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257) ~[client-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.IllegalArgumentException: Amount may not be negative, but it is: -2
	at net.fabricmc.fabric.api.transfer.v1.storage.StoragePreconditions.notNegative(StoragePreconditions.java:50) ~[fabric-transfer-api-v1-2.1.6+413cbbc790-e9d4a58328abaa4.jar:?]
```

The other one might be the same.

--- a/fabric/src/main/java/owmii/powah/fabric/block/FabricCableTile.java
+++ b/fabric/src/main/java/owmii/powah/fabric/block/FabricCableTile.java
@@ -35,7 +35,7 @@ public class FabricCableTile extends CableTile {
             if (amount <= 0)
                 break;
             if (!cable.energySides.isEmpty() && cable.isActive()) {
-                received += ((FabricCableTile) cable).pushEnergy(maxReceive, transaction, direction, this);
+                received += ((FabricCableTile) cable).pushEnergy(amount, transaction, direction, this);
             }
         }
 

--- a/forge/src/main/java/owmii/powah/forge/block/ForgeCableTile.java
+++ b/forge/src/main/java/owmii/powah/forge/block/ForgeCableTile.java
@@ -31,7 +31,7 @@ public class ForgeCableTile extends CableTile {
             if (amount <= 0)
                 break;
             if (!cable.energySides.isEmpty() && cable.isActive()) {
-                received += ((ForgeCableTile) cable).pushEnergy(maxReceive, simulate, direction, this);
+                received += ((ForgeCableTile) cable).pushEnergy(amount, simulate, direction, this);
             }
         }
 

